reporters/ClickhouseReporter.js

```diff
@@ -95,47 +95,55 @@ const TABLE_DEFINITIONS = [
     PRIMARY KEY(crawlId, pageId, targetId)`,
 ];
 
+/**
+ * @param {string | string[]} args
+ */
+function santizeCallArgs(args) {
+    // in some cases call args have been stringified, so unwrap that first.
+    const argsArray = typeof args === 'string'  ? JSON.parse(args) : (args || []);
```

Comment:
```suggestion
    const argsArray = typeof args === 'string' ? JSON.parse(args) : (args || []);
```

can you please double check why eslint didn't catch this?

Reply:
Looks like `space-infix-ops` is off in our eslintrc. Shall I turn it on?